ext/django_stubs_ext/aliases.py

```diff
@@ -6,7 +6,7 @@
     from django.utils.functional import _StrPromise as StrPromise
 
     QuerySetAny = _QuerySetAny
-    ValuesQuerySet = _QuerySet[_T, _Row]
+    ValuesQuerySet = _QuerySet[_T, _Row]  # TODO think about this!
```

Comment:
Reminder for self.

Reply:
Tested and this works fine, even though `query.pyi` now uses `_Model` TypeVar instead of `_T` in the `QuerySet` class (`_T` still exists).

It's best to keep these aliases as is for now. Since the dependency constraint between `django-stubs` and `django-stubs-ext` only goes one way, users may be running e.g. `django-stubs-ext 5.0.1` and `django-stubs 5.0.0` together.

We can simplify these aliases more a few releases down the line.